### PR TITLE
[DO NOT MERGE] archive(8.1) exclude: remove versions 8.1 and 3.9.0, part 2: chonky boi

### PR DESCRIPTION
**Pull request by pepopowitz**
This PR contains a ton of removed files! Viewing "Files changed" will crash your browser.

It targets the branch from #3769, since that one should merge first.

**BLOCKED**

- [ ] #3769 should be merged first.

## Description

Part of #3731.

Deletes the 8.1/3.9.0 content from the docs. This is the final step in archiving this unsupported version.

Since it's hard to view in the browser, these are the commands I ran for this PR: 

```sh
rm -rf ./versioned_docs/version-8.1/
rm ./versioned_sidebars/version-8.1-sidebars.json
rm -rf ./optimize_versioned_docs/version-3.9.0/
rm ./optimize_versioned_sidebars/version-3.9.0-sidebars.json
```

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] There is **no urgency** with this change and can be released at any time.
- [ ] In the next couple days, preferably, so I can complete this archival work.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineerring DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer.

- [x] My changes require a [docs infrastructure review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process).

### CODE REVIEW

**Review by akeller — APPROVED** [review]
Hand wavy review because that's all I can do (can barely type TBH)

**Comment by pepopowitz** [comment]
Sorry to do this @akeller but could I get another approval on this? I merged the other related PR, resulting in this one changing targets to `main` (as expected)...and it dismissed your approval.

For what it's worth, if approving through the UI is completely unbearable due to the massive quantity of changes, the GitHub CLI provides a method for reviewing/approving from your terminal: https://cli.github.com/manual/gh_pr_review

**Review by akeller — APPROVED** [review]

**Comment by pepopowitz** [comment]
Merging in a red state -- this is as green as it's going to get, because the massive quantity of changes breaks the two failing checks.

**Pull request merged**
